packages/wallet/tsconfig.prod.cjs.json

```diff
@@ -2,8 +2,7 @@
   "extends": "../../config/tsconfig.prod.cjs.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "dist/cjs",
-    "composite": true
+    "outDir": "dist/cjs"
```

Comment:
Can we also move these `outDir` commands to the `config/cli/tsconfig.prod.*.json` files?